src/lib.rs

```diff
@@ -119,7 +119,7 @@
 //! assert_eq!(decoded_enr.ip4(), Some("192.168.0.1".parse().unwrap()));
 //! assert_eq!(decoded_enr.id(), Some("v4".into()));
 //! assert_eq!(decoded_enr.tcp4(), Some(8001));
-//! assert_eq!(decoded_enr.get("custom_key"), Some(vec![0,0,1].as_slice()));
+//! assert_eq!(decoded_enr.get("custom_key").as_ref().map(AsRef::as_ref), Some(vec![0,0,1]).as_deref());
```

Comment:
could you check if this can be simplified?

Reply:
Sorry I couldn't find a way to make it simple because the first part (```.get("custom_key")```) is ```Option<Bytes>``` and the second part (```Some(vec![0, 0, 1])```) is ```Option<Vec<{integer}>>```. I couldn't find any relation with these two. I'm waiting your suggestions.